README.md

```diff
@@ -72,7 +72,7 @@ name. That seems to be the fairest way to arrange this table.
 | MATLAB | [mlint](https://www.mathworks.com/help/matlab/ref/mlint.html) |
 | OCaml | [merlin](https://github.com/the-lambda-church/merlin) see `:help ale-integration-ocaml-merlin` for configuration instructions
 | Perl | [perl -c](https://perl.org/), [perl-critic](https://metacpan.org/pod/Perl::Critic) |
-| PHP | [php -l](https://secure.php.net/), [phpcs](https://github.com/squizlabs/PHP_CodeSniffer) |
+| PHP | [php -l](https://secure.php.net/), [phpcs](https://github.com/squizlabs/PHP_CodeSniffer), [hh_client](http://hacklang.org/) |
```

Comment:
Should we maybe name the linter itself `hh_client`, or maybe use the name `hack` here instead, and in the docs? It would be nice to make the name easier to guess for people who want to turn it on or off specifically. I'd also move it to the front of the list, as everything is supposed to be sorted by name here.